doc/examples/tls.rst

```diff
@@ -147,28 +147,29 @@ Client certificates
 ...................
 
 PyMongo can be configured to present a client certificate using the
-`ssl_certfile` option::
+``tlsCertificateKeyFile`` option::
 
   >>> client = pymongo.MongoClient('example.com',
-  ...                              ssl=True,
-  ...                              ssl_certfile='/path/to/client.pem')
+  ...                              tls=True,
+  ...                              tlsCertificateKeyFile='/path/to/client.pem')
 
 If the private key for the client certificate is stored in a separate file use
-the `ssl_keyfile` option::
```

Comment:
I've kept in `ssl_keyfile` because we currently still support it. https://jira.mongodb.org/browse/PYTHON-2396 will figure out if we want to remove it in PyMongo 4.0.